include/iotc_types.h

```diff
@@ -30,66 +30,59 @@ extern "C" {
+ * @name iotc_timed_task_handle_t
+ * @brief Timed task handle to identify timed tasks.
  */
 typedef int32_t iotc_timed_task_handle_t;
 
 /**
- * @name    iotc_user_task_callback_t
- * @brief   User custom action which can be passed to the
- * iotc_schedule_timed_task to have it executed in the given time intervals.
- *
- * @param [in]  in_context_handle the context handle provided to
- * iotc_schedule_timed_task.
- * @param [in]  timed_task_handle is the handle that identifies timed task and
- * it allows to manipulate the timed task from within the callback e.g. to
- * cancel the task if it's no longer needed.
- * @param [in]  user_data is the data provided to iotc_schedule_timed_task.
+ * @name iotc_user_task_callback_t
+ * @brief Custom callback for timed tasks.
+ * @detailed This is a user-defined custom callback. Pass the callback to
+ * <code>iotc_schedule_timed_task()</code> repeatedly execute the callback at
+ * the given interval.
+ *
+ * @param [in] in_context_handle the context handle provided to
+ * <code>iotc_schedule_timed_task()</code>.
+ * @param [in] timed_task_handle the handle that identifies the timed task.
+ * @param [in] user_data the data provided to <code>iotc_schedule_timed_task()</code>.
  */
 typedef void(iotc_user_task_callback_t)(
     const iotc_context_handle_t context_handle,
     const iotc_timed_task_handle_t timed_task_handle, void* user_data);
 
 /**
- * @name    iotc_user_callback_t
- * @brief   Some API functions notify the application about the result through
- *          callbacks, these callbacks can be passed as iotc_user_callback_t to
- *          the API.
- *
- * @param [in]  in_context_handle the context handle which was provided to the
- *              original API call.
- * @param [in]  data contains API specific information. Before usage it should
- *              be cast to specific type depending on the original API call.
- *              E.g. iotc_connect's callback receives iotc_connection_data_t*
- *              in the data attribute. Thus its usage is as follows:
- *              iotc_connection_data_t* conn_data =
- *              (iotc_connection_data_t*)data;
- * @param [in]  state is the result of the API call. IOTC_STATE_OK in case of
- *              success. For other error codes please see the IoTC User Guide
- *              or Examples.
+ * @name     iotc_user_callback_t
+ * @brief    Custom callback.
+ * @detailed Callbacks are passed to <code>iotc_user_callback_t</code> Callbacks
+ *           notify the client application after a function executes.
+ *
+ * @param [in] in_context_handle the context handle provided to the original
+ * API call.
+ * @param [in] data API-specific information. Cast to a specific type depending
+ * on the original API call. For example, the <code>iotc_connect()</code>
+ * callback returns the iotc_connection_data_t* type so <code>iotc_connection_data_t* conn_data = (iotc_connection_data_t*)data;</code>.
```

Comment:
I think that the format code script will break this into muiltiple line. We have a max line length of 80 or  90 I believe.